src/server.ts

```diff
@@ -17,5 +17,12 @@ export function makeApp(db: Db): core.Express {
     response.render("index");
   });
 
+  app.get("/platform", (request, response) => {
+    response.render("platform");
```

Comment:
```suggestion
    response.render("platforms");
```
Je reajouterais un s ici aussi